kiwi/builder/disk.py

```diff
@@ -251,17 +251,19 @@ def create_disk(self):
         )
 
         # create the bootloader instance
-        self.bootloader_config = BootLoaderConfig.new(
-            self.bootloader, self.xml_state, root_dir=self.root_dir,
-            boot_dir=self.root_dir, custom_args={
-                'targetbase':
-                    self.loop_provider.get_device(),
-                'grub_directory_name':
-                    Defaults.get_grub_boot_directory_name(self.root_dir),
-                'boot_is_crypto':
-                    self.boot_is_crypto
-            }
-        )
+        self.bootloader_config = None
```

Comment:
One comment on this. I think it's not needed to have self.bootloader_config initialized as None. All code accessing bootloader_config is only reached if if ```self.bootloader != 'custom``` and in that case we never want to run into the situation that self.bootloader_config is None. I think we rather want the error of the python stack trace in this case.

Reply:
Removed in the updated patch.